package.json

```diff
@@ -3,22 +3,22 @@
   "version": "1.0.23",
   "description": "Query less exploration of log data stored in Loki",
   "scripts": {
-    "build": "webpack -c ./webpack.config.ts --env production",
-    "dev": "webpack -w -c ./webpack.config.ts --env development",
+    "build": "webpack -c ./.config/webpack/webpack.config.ts --env production",
```

Comment:
go home github actions, you're drunk

Reply:
Haha yeah they do this every time, I will revert it 👍